src/commercetools/helpers.py

```diff
@@ -149,6 +150,34 @@ def get_schema(self, name):
         schema.ordered = getattr(self.parent, "ordered", False)
         return schema
 
+    def get_schema_class(self, name):
+        """Get schema class for this field.
+
+        Due do lazy loading imports keep retrying.
+        """
+        tries = 0
+        max_tries = 50  # more than the amount of schema .py files
+        last_import_error: typing.Optional[str] = None
+        while tries < max_tries:
```

Comment:
Why do we need retrires?

Reply:
As mentioned on Slack, if you have an Order object,  it might rely on some schema in _cart, and schemas in other files, so you have to retry until you have them all.

We probably have to also use this logic for Nested fields, if you agree with this approach?